toolkits/global/packages/global-forms/README.md

```diff
@@ -1,12 +1,26 @@
 # Global Forms
 
-This component includes a number of form fields and related templates.
+The forms component provides a range of form elements and inputs that allow users to enter information. It also includes error messages. It is designed to make it as simple as possible to create an HTML form.
 
-The component does not include any JavaScript. States (such as an invalid/error state) are defined at a data level. If you're using client-side processing, you might benefit from compiling the form’s handlebars template in the browser.
+## When to use this component
+Use the forms component when users need to submit information or data. This could be in order to complete a task or as part of a service we provide. For example, when an author needs to enter the title of their manuscript when submitting to a journal.
 
-## Usage
 
-First, include the necessary Sass files in your project.
+## How it works
+The forms component does not include any JavaScript. States (such as an invalid/error state) are defined at a data level. If you're using client-side processing, you might benefit from compiling the form’s handlebars template in the browser.
+
+- Data structure
+- Interdependencies
+- Registering partials
```

Comment:
As someone that reads this doc for the first time, I find it hard to understand what Interdependencies in this case means, maybe we could say a bit more about those 3 points?